modules/health_quest/app/services/health_quest/base_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  class BaseService < Common::Client::Base
+    include Common::Client::Concerns::Monitoring
+    include SentryLogging
+
+    # LJG NOTE: statsd key for health quest different than VAOS
```

Comment:
what does `LJG NOTE` mean?

Reply:
It's my initials. I don't have to put these comments in the code with my initials, but it makes it easy for me to grep these notes for my initials.

Reply:
I think it's better to keep notes like this out of the code when we can, I think everything we merge should be meaningful to everyone looking at the code

Reply:
I removed that comment